src/components/chips/js/chipsDirective.js

```diff
@@ -62,6 +62,8 @@
    *  </span>
    *
    * @param {string=|object=} ng-model A model to bind the list of items to
+   * @param {expression=} md-on-select An expression to be run each time a new item got selected
```

Comment:
LGTM, but one last change: Can we make these say

> An expression which will be called when a chips is selected. The `$chip` and `$index` variables are made available in the expression.

Similar for the deselect.